roles/gitlab/tasks/install.yml

```diff
@@ -122,16 +122,30 @@
 - name: "Install GitLab and add helper to detect failed reconfigures"
   block:
 
+    - name: Unhold GitLab if it does not contain version info
+      ansible.builtin.dpkg_selections:
+        name: "{{ gitlab_package_name }}"
+        selection: install
+      when: "'=' not in gitlab_package_name"
+
```

Comment:
This will break on non debian-like systems.

Which seem to be supported: https://github.com/adfinis/ansible-collection-gitlab/blob/main/roles/gitlab/tasks/install.yml#L67-81